c++/src/kj/miniposix.h

```diff
@@ -44,6 +44,8 @@
 #include <sys/types.h>
 #endif
 
+#include <sys/uio.h>
```

Comment:
Sorry but also this `#include` should not apply on Windows, so you'll need to place it in the "else" block of the `#if _WIN32` above.

(Apologies for the slow turn-around. Swamped as always.)